src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -44,6 +44,8 @@ case class ExecutionServiceLogs(
  logs: Map[String, Seq[Map[String,String]]]
 )
 
+case class ExecutionServiceWorkflowOptions(jes_gcs_root: String)
```

Comment:
This refers to a specific option instead of execution service workflow options more generally, so it should be named for that option.  Maybe `ExecutionServiceOutputDestination`.  While there is only one option available now, there could be more in the future.